src/query/interpreter.cpp

```diff
@@ -721,7 +725,7 @@ Callback HandleStreamQuery(StreamQuery *stream_query, const Parameters &paramete
   Callback callback;
   switch (stream_query->action_) {
     case StreamQuery::Action::CREATE_STREAM: {
-      EventCounter::IncrementCounter(EventCounter::StreamsCreated);
+      memgraph::metrics::IncrementCounter(memgraph::metrics::StreamsCreated);
```

Comment:
I think this method GetKafkaCreateCallback can throw error. Since this is callback, maybe it makes sense to move this inside functions?